addon/HomepageStudy.jsm

```diff
@@ -74,16 +74,25 @@ function getHomepage() {
 var HomepageStudy = {
   /**
    * Returns the value encoded by RAPPOR or null if the homepage can't be obtained.
+   * The two last arguments are not used, but must be declared for consistence with Simulator.jsm
    * @param {string} studyName - Name of the study.
+   * @param {boolean} isSimulation - Boolean indicating whether the execution is for a simulation.
+   * @param {string} rapporPath - Path where the RAPPOR simulator is located.
    *
    * @returns the encoded value returned by RAPPOR or null if the eTLD+1 can't be obtained.
    */
-  reportValue(studyName) {
+  reportValue(studyName, isSimulation = false, rapporPath = null) {
     let eTLDHomepage = getHomepage();
     if (!eTLDHomepage) {
-      return null;
+       return null;
     }
-    return TelemetryRappor.createReport(studyName, eTLDHomepage, {filterSize: 16, numHashFunctions: 2,
-                                        cohorts: 100, f: 0.0, p: 0.35, q: 0.65});
+    let report = TelemetryRappor.createReport(studyName, eTLDHomepage,
+                                             {filterSize: 16, numHashFunctions: 2, cohorts: 100, f: 0.0, p: 0.35, q: 0.65},
+                                             Ci.nsICryptoHash.MD5);
+    return {
```

Comment:
Do you really need that? The name of the properties are the same, so just return at line 89.

Reply:
Done